pkg/kubernetes/storageclass/v1alpha1/storageclass.go

```diff
@@ -0,0 +1,109 @@
+package v1alpha1
+
+import (
+	storagev1 "k8s.io/api/storage/v1"
+)
+
+// StorageClass holds the api's Storageclass object
+type StorageClass struct {
+	object *storagev1.StorageClass
+}
+
+// StorageClassList holds the list of StorageClass instances
+type StorageClassList struct {
+	items []*StorageClass
+}
+
+// Predicate defines an abstraction
+// to determine conditional checks
+// against the provided StorageClass instance
+type Predicate func(*StorageClass) bool
+
+// predicateList holds the list of predicates
+type predicateList []Predicate
+
+// ListBuilder enables building an instance of StorageClassList
+type ListBuilder struct {
+	list    *StorageClassList
+	filters predicateList
+}
+
+// WithAPIList builds the list of StorageClass
+// instances based on the provided
+// StorageClass list
+func (b *ListBuilder) WithAPIList(storageclasses *storagev1.StorageClassList) *ListBuilder {
+	if storageclasses == nil {
+		return b
+	}
+	b.WithAPIObject(storageclasses.Items...)
+	return b
+}
+
+// WithObject builds the list of StorageClass instances based on the provided
+// StorageClass list instance
+func (b *ListBuilder) WithObject(StorageClasses ...*StorageClass) *ListBuilder {
```

Comment:
name of argument can be `storageClasses` i.e. small letter

Reply:
Updated to `scs`